addons/website/static/src/scss/primary_variables.scss

```diff
@@ -2024,6 +2024,7 @@ $o-base-website-values-palette: (
     'input-border-radius': null, // Default to BS
     'input-border-radius-sm': null, // Default to BS
     'input-border-radius-lg': null, // Default to BS
+    'input-background-color': null,
```

Comment:
Since it is a color, add this in the color palette. Agreed, it may be annoying and could be reviewed in the future. Note that this may be cool functionally too as it would allow you to choose like "gray-500" for the color and have the input color follow the gray-500 changes afterwards automatically (although you may need to configure the colorpicker to enable that, not sure anymore).

Reply:
> Since it is a color, add this in the color palette.

To be sure; do you mean adding an `input-background-color` key set to `#fff` in any default palette?

Reply:
No, adding the default "null" value in the `$o-base-color-palette`, not `$o-base-website-values-palette`